### PR TITLE
Add missing validation messages

### PR DESCRIPTION
**Pull request by ethax-ross**
- Add missing validation messages

Some attribute validation rules didn't have corresponding custom error messages; this introduces them to the translations file.

- Remove redundant validation checks

These attributes are not set by the user; instead they are set by the model internally. If these validation checks ever failed we would end up showing an error message to the user that they could do nothing about, and we would not be alerted to a bug in the application code.

Instead, by removing the validation checks it will enable the candidate to get to the end of the submission and they will then receive an error when the form attempts to submit to the API. We will also be alerted about this, which feels like a better outcome/experience for both the candidate and ourselves.

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -108,10 +108,17 @@ en:
           attributes:
             first_name:
               blank: "You need to enter your first name"
+              too_long: "You have entered too many characters - you need to enter your first name"
             last_name:
               blank: "You need to enter your surname"
+              too_long: "You have entered too few characters - you need to enter your surname"
```
Comment (peteryates) [inline config/locales/en.yml:114]: I think this should be `too many`

**Review by peteryates — COMMENTED** [review]
Looks good but there's a typo in the `last_name.too_long` error message.

File: config/locales/en.yml (hunk 1 above)
Comment (ethax-ross) [reply config/locales/en.yml:114]: Yep, well spotted!

**Review by peteryates — APPROVED** [review]
:+1:

**Pull request merged**